src/rethinkdb-orm/connection.cr

```diff
@@ -46,7 +46,9 @@ module RethinkORM
     # The block defined query is run and raw results returned.
     def self.raw(**options)
       query = yield R
-      query.run(self.db, **options)
+      Retriable.retry(max_attempts: settings.retry_attempts || 10, on: IO::Error) do
```

Comment:
Can the default for retry attempts be specified in [settings](https://github.com/spider-gazelle/rethinkdb-orm/blob/100537da0a6896364313ef46da717227bd226ec3/src/rethinkdb-orm/settings.cr#L12) to match how other defaults are handled?

Reply:
technically that setting is for the driver's reconnection logic.
i could add another setting for `query_retry_attempts`

Reply:
Yep, that's probably the neat approach.